src/main/java/ui/RepositorySelector.java

```diff
@@ -49,6 +52,17 @@ private void setupComboBox() {
                 onValueChangeCallback.accept(repoId);
             }
         });
+        
+        // remove focus from the repo selector
+        comboBox.addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+            event.consume();
+            if (event.getCode() == KeyboardShortcuts.REMOVE_FOCUS) {
+                if (ui.isTestMode()) {
```

Comment:
plus we do the same thing [here] (https://github.com/HubTurbo/HubTurbo/blob/master/src/main/java/ui/listpanel/ListPanel.java#L328)